src/category_theory/sites/sheaf.lean

```diff
@@ -484,7 +528,7 @@ lemma is_sheaf_for.valid_glue
 t.is_amalgamation hx f Hf
 
 /-- C2.1.3 in [Elephant] -/
-lemma is_sheaf_for_iff_generate :
+lemma is_sheaf_for_iff_generate (R : presieve X) :
```

Comment:
This change makes it easier to use this as a rewrite lemma especially when there are multiple instances of `is_sheaf_for` in the goal, since in theory you could keep applying it infinitely: `is_sheaf_for P R -> is_sheaf_for P (generate R) -> is_sheaf_for P (generate (generate R)) -> ...`, but with the name you can restrict to just the first